README.md

```diff
@@ -200,6 +200,30 @@ $ ddg-screen-diff ia products foo bar baz qux
 
 Passing `prod` or `production` as a hostname will alias it to the production server.
 
+Actions
+---
+
+With any screenshot command you can pass the `--action`/`-a` option to specify the name of an action to run before taking the screenshot. These are defined in a folder specified under `actionDir` in your config.
+
+Here's an example action file called `click_button_and_mouse_out.json`:
+
+```json
+[
+    { "action": "click", target: "#button_id" },
+    { "action": "mouseMove", target: { x: 10, y: 10 }
+]
+```
+
+You invoke that by running:
+
+```
+$ ddg-screen-diff search "foo" --action click_button_and_mouse_out
+```
+
+Currently supported values for `action` are `click`, `mouseMove`, `mouseUp` and `mouseDown`.
```

Comment:
Might be worth adding a `wait` action? I could see cases we'll want to click something, wait 300ms for a transition to finish, then take the screenshot?